stats/DataAggregation.md

```diff
@@ -0,0 +1,67 @@
+# Data Aggregation API Overview
+The stats library allows users to define measurements how data are aggregated and broken down.
```

Comment:
I'm not sure I understand this sentence.  Should "measurements" be replaced with "view"?

Reply:
Good catch. Fixed.